slapo/autotune/tune.py

```diff
@@ -282,7 +282,7 @@ def run_training_script(args, tuneable_cfg):
     # Set all tunable parameters as environment variables.
     env = " ".join(f"{k}={v}" for k, v in tuneable_cfg.items())
 
-    cmd = f"DEBUG=1 {env} python3 {args.training_script} {train_script_args}"
+    cmd = f"{env} python3 {args.training_script} {train_script_args}"
```

Comment:
Can we pass in the debug flag here for auto-tuning?

Reply:
Note that DEBUG flag is removed completely in this PR. Instead, we could probably disable consolidation using the flag in the tuning script.

Reply:
I see